### ISSUE TITLE
Capture Zones count AI even when "Cannot Capture" is toggled

### ISSUE THREAD

**Issue by JudahHarvey** [PiZZAD0X/MMFW#13]
AI of a certain faction can currently capture zones even when AI "cannot capture" is ticked for that side

**Comment by PiZZAD0X** [comment]
Do they actually capture it or does it simply say they are capturing it? Found where it could display them in the process of capturing it but couldn't find where they would actually capture it

**Comment by JudahHarvey** [comment]
I don't believe the capturing actually fires, I have a trigger checking for the var, it just gets more and more red as they capture but doesn't seem to actually cap

**Comment by JudahHarvey** [comment]
I mean maybe this should just be left up to the mission maker to disable the messages so it doesn't display
